tests/test_api_c3voc.py

```diff
@@ -241,6 +244,81 @@ def test_request_voctoweb_clears_voctoweb(client, app, db, proposal):
             "voctoweb": {
                 "enabled": True,
                 "frontend_url": "",
+                "thumb_path": "",
+            },
+            "youtube": {
+                "enabled": False,
+            },
+        },
+    )
+    assert rv.status_code == 204
+
+    proposal = Proposal.query.get(proposal.id)
+    assert proposal.c3voc_url is None
+
+
+def test_update_thumbnail_with_path(client, app, db, proposal):
+    app.config.update(
+        {
+            "VIDEO_API_KEY": "api-key",
+        }
+    )
+
+    clean_proposal(db, proposal)
```

Comment:
Just a note that I'm attempting two cleanup refactorings locally, although running into some challenges:

  * I think we could move most of the `clean_proposal` -- which seems a mixture of unit test setup and teadown logic -- into the fixture itself, by using [`yield`](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended) instead of `return`.
  * We could _probably_ move the `VIDEO_API_KEY` into the `config/test.cfg` file -- although I guess even so we need to override it for one or two of the tests.

Reply:
regarding `clean_proposal()`: yes. personally i'm a fan of starting every test on the same clean slate, so we should try to implement that here as well.

I have no opinions about the VIDEO_API_KEY.

Reply:
There's a relatively nice pattern in the Sphinx unit tests where tests can have a decorator applied to them that configures some settings temporarily for the duration of a test using a dictionary object.  It keeps the declarative / test setup near the test case, but is a bit more concise (writing to `app.config` within test cases seems strange).

Where I'm stuck with the fixture teardown refactoring is that some errors are occurring when I run the tests in random order.  So something about my adjusted teardown logic must be incorrect..

I don't think any of this is essential, but I'd like to try to provide a follow-up pull request if I get it working.

Reply:
Possible test adjustments opened at #1776 - the fixture teardown problem was because I hadn't noticed the `module`-level scoping of the fixture.